app/controllers/registration_commands_controller.rb

```diff
@@ -24,7 +24,9 @@ def operatingsystem_template
       return
     end
 
-    render json: { template: { name: template.name, path: edit_provisioning_template_path(template) } }
+    ptemplate = template.provisioning_template
```

Comment:
Shouldn't we rename the `template` to `os_default` to make it more clear?
I'd even love if we don't use result of a `<method>?` because it should in theory always return boolean.
But that is just me nitpicking, but naming it template is missleading, I needed to go to the `has_default_template?` to get why you do `template.provisioning_template`, naming it differently would help to understand it IMHO.